boards/arm/disco_l475_iot1/Kconfig.board

```diff
@@ -8,5 +8,6 @@
 config BOARD_DISCO_L475_IOT1
 	bool "Discovery IoT L475 Development Board"
 	select HAS_DTS_I2C_DEVICE
+	select HAS_DTS_SPI
```

Comment:
Why should we enable HAS_DTS_SPI per board? When SPI_STM32 is enabled, it could select HAS_DTS_SPI in Kconfig.